modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -23,6 +23,13 @@ You run a DPDK checkup by performing the following steps:
 * You have installed the OpenShift CLI (`oc`).
 * You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
 
++
+[WARNING]
```

Comment:
Would suggest changing the admonition type to `[IMPORTANT]`. According to our style guide, `[WARNING]` is to be used for relaying information about potential system damage, data loss, or a support-related issue.

Reply:
Done.